python/cudf/cudf/tests/test_dtypes.py

```diff
@@ -341,7 +341,6 @@ def test_dtype(in_dtype, expect):
         np.complex128,
         complex,
         "S",
-        "a",
```

Comment:
Just avoids an error `"a"` has been removed, and seems not particularly important here.